config/routes.rb

```diff
@@ -41,6 +41,13 @@
     end
   end
 
+  # Custom route for looking for member
```

Comment:
If this functionality is added to the edit page then this route can also be removed.